src/ac_power_flow_jacobian.jl

```diff
@@ -14,28 +14,32 @@ function (Jacobianf::PolarPowerFlowJacobian)(
     return
 end
 
-function PolarPowerFlowJacobian(data::ACPowerFlowData, x0::Vector{Float64})
-    j0 = _create_jacobian_matrix_structure(data)
+function PolarPowerFlowJacobian(
+    data::ACPowerFlowData,
+    x0::Vector{Float64};
+    time_step::Int64 = 1,
+)
+    j0 = _create_jacobian_matrix_structure(data; time_step = time_step)
     F0 =
         (J::SparseArrays.SparseMatrixCSC{Float64, Int32}, x::Vector{Float64}) ->
-            jsp!(J, x, data)
+            jsp!(J, x, data; time_step = time_step)
     F0(j0, x0)
     return PolarPowerFlowJacobian(F0, j0)
 end
 
-function _create_jacobian_matrix_structure(data::ACPowerFlowData)
+function _create_jacobian_matrix_structure(data::ACPowerFlowData; time_step::Int64 = 1)
```

Comment:
why is this a kwarg, this function is internal and can take a regular argument, kwargs have big performance costs.